Automattic/wp-calypso
# Update border radius properties for better parity with Gutenberg

#### Changes proposed in this Pull Request

* Add 2px border radius to form inputs (textarea, etc.)
* Remove border radius from non-actionable elements (ie. notices).
* Reduce border radius to 2px across the board, with the exception of circles and pill-button styles
* All of the above better aligns Calypso styles with Gutenberg styles

**Visuals (not exhaustive)**

Before | After
------- | -------
<img width="273" alt="before" src="https://user-images.githubusercontent.com/2124984/90818440-83420780-e2fc-11ea-8078-46122ea7ec58.png"> | <img width="272" alt="after" src="https://user-images.githubusercontent.com/2124984/90818438-82a97100-e2fc-11ea-836e-2bfe6b4b10e8.png">
<img width="433" alt="before2" src="https://user-images.githubusercontent.com/2124984/90649775-1568e380-e209-11ea-864b-22f89e0b9847.png"> | <img width="420" alt="after2" src="https://user-images.githubusercontent.com/2124984/90649768-14d04d00-e209-11ea-9a59-168242a189b5.png">
<img width="415" alt="before3" src="https://user-images.githubusercontent.com/2124984/90649776-1568e380-e209-11ea-869c-0b2f478c611d.png"> | <img width="415" alt="after3" src="https://user-images.githubusercontent.com/2124984/90649770-14d04d00-e209-11ea-837d-8adb029e8824.png">
<img width="236" alt="before4" src="https://user-images.githubusercontent.com/2124984/90649778-1568e380-e209-11ea-9d41-be9fddf4c47e.png"> | <img width="238" alt="after4" src="https://user-images.githubusercontent.com/2124984/90649771-14d04d00-e209-11ea-85bf-2ec42bee15c2.png">
<img width="1072" alt="Screen Shot 2020-08-20 at 3 49 35 PM" src="https://user-images.githubusercontent.com/2124984/90822349-4b3dc300-e302-11ea-91e1-9e8e4a972d99.png"> | <img width="1068" alt="Screen Shot 2020-08-20 at 3 49 16 PM" src="https://user-images.githubusercontent.com/2124984/90822358-4f69e080-e302-11ea-9256-95154d715359.png">


#### Testing instructions

* Switch to this PR
* Note visual changes in border radii across Calypso; form fields, buttons, popovers should have 2px rounded corners.

See #44883
